<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kmuiClass</class>
 <widget class="QMainWindow" name="kmuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>607</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KMMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="left_part" native="true">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>200</width>
      <height>381</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="anchor_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>锚点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="related_entries_button">
         <property name="text">
          <string>关联</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tag_button">
         <property name="text">
          <string>标签</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="synopsis_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>大纲</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="left_tab_widget"/>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tab_widget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>6</y>
      <width>500</width>
      <height>321</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #f0f0f0;
}

QTabBar::tab {
    padding: 8px 10px; 
    background: #e0e0e0;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 80px;
    border: 1px solid #cccccc;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: #fcfcfc;
    border-color: rgb(228, 228, 228);
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::TabPosition::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::TabShape::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::TextElideMode::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="act_create_entry"/>
    <addaction name="act_delete_entry"/>
    <addaction name="act_set_current_entry_as_anchor"/>
    <addaction name="separator"/>
    <addaction name="act_create_new_knowledge_library"/>
    <addaction name="act_open_knowledge_library"/>
    <addaction name="act_save_kl"/>
    <addaction name="act_recent_knowledge_library"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="act_undo"/>
    <addaction name="act_redo"/>
    <addaction name="separator"/>
    <addaction name="act_add_text_block"/>
    <addaction name="act_add_code_block"/>
    <addaction name="act_add_image_block"/>
    <addaction name="act_add_header_block"/>
    <addaction name="separator"/>
    <addaction name="act_point_out"/>
   </widget>
   <widget class="QMenu" name="menu_format">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="act_set_type_normal"/>
    <addaction name="act_set_type_code"/>
    <addaction name="act_set_type_link"/>
    <addaction name="separator"/>
    <addaction name="act_bold"/>
    <addaction name="act_italic"/>
    <addaction name="act_underline"/>
    <addaction name="act_strike"/>
    <addaction name="separator"/>
    <addaction name="act_remove_bold"/>
    <addaction name="act_remove_italic"/>
    <addaction name="act_remove_underline"/>
    <addaction name="act_remove_strike"/>
   </widget>
   <widget class="QMenu" name="menu_tag">
    <property name="title">
     <string>标签</string>
    </property>
    <addaction name="act_manage_tag"/>
   </widget>
   <widget class="QMenu" name="menu_search">
    <property name="title">
     <string>搜索</string>
    </property>
    <addaction name="act_search_entry"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_format"/>
   <addaction name="menu_tag"/>
   <addaction name="menu_search"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_create_entry">
   <property name="text">
    <string>新建词条</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="act_delete_entry">
   <property name="text">
    <string>删除当前词条</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="act_save_kl">
   <property name="text">
    <string>保存知识库</string>
   </property>
   <property name="toolTip">
    <string>保存知识库</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="act_bold">
   <property name="text">
    <string>加粗</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="act_italic">
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="act_underline">
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="act_strike">
   <property name="text">
    <string>删除线</string>
   </property>
  </action>
  <action name="act_search_entry">
   <property name="text">
    <string>搜索词条</string>
   </property>
  </action>
  <action name="act_manage_tag">
   <property name="text">
    <string>管理标签</string>
   </property>
  </action>
  <action name="act_add_text_block">
   <property name="text">
    <string>添加文本块</string>
   </property>
  </action>
  <action name="act_add_code_block">
   <property name="text">
    <string>添加代码块</string>
   </property>
  </action>
  <action name="act_add_image_block">
   <property name="text">
    <string>添加图片块</string>
   </property>
  </action>
  <action name="act_add_header_block">
   <property name="text">
    <string>添加标题块</string>
   </property>
  </action>
  <action name="act_set_current_entry_as_anchor">
   <property name="text">
    <string>将当前词条设为锚点</string>
   </property>
  </action>
  <action name="act_point_out">
   <property name="text">
    <string>添加关联指出</string>
   </property>
  </action>
  <action name="act_set_type_normal">
   <property name="text">
    <string>普通文本</string>
   </property>
  </action>
  <action name="act_set_type_code">
   <property name="text">
    <string>内联代码</string>
   </property>
  </action>
  <action name="act_set_type_link">
   <property name="text">
    <string>链接</string>
   </property>
  </action>
  <action name="act_remove_bold">
   <property name="text">
    <string>移除加粗</string>
   </property>
  </action>
  <action name="act_remove_italic">
   <property name="text">
    <string>移除斜体</string>
   </property>
  </action>
  <action name="act_remove_underline">
   <property name="text">
    <string>移除下划线</string>
   </property>
  </action>
  <action name="act_remove_strike">
   <property name="text">
    <string>移除删除线</string>
   </property>
  </action>
  <action name="act_open_knowledge_library">
   <property name="text">
    <string>打开知识库</string>
   </property>
  </action>
  <action name="act_create_new_knowledge_library">
   <property name="text">
    <string>新建知识库</string>
   </property>
  </action>
  <action name="act_recent_knowledge_library">
   <property name="text">
    <string>查看历史知识库</string>
   </property>
  </action>
  <action name="act_undo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="act_redo">
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../KnowledgeManager.qrc"/>
 </resources>
 <connections/>
</ui>
